src/org/anjocaido/groupmanager/dataholder/worlds/ChildMirrors.java

```diff
@@ -19,8 +19,8 @@
 	 * 
 	 * Mirror shows the same data as parent.
 	 */
-	private Map<String, String> groupsMirror = new TreeMap<>();
-	private Map<String, String> usersMirror = new TreeMap<>();
+	private final Map<String, String> groupsMirror = new TreeMap<>();
+	private final Map<String, String> usersMirror = new TreeMap<>();

```

Comment:
I'll check tomorrow, but in general, reloads are near guaranteed to cause problems and server admins are advised to avoid reloading as much as possible